### PR TITLE
Repo: add Vulnerability Response Process

### PR DESCRIPTION
**Pull request by anonimal**
As discussed in Kovri meetings,IRC, and https://github.com/monero-project/monero/pull/1995: here is a formalized VRP. This VRP will aid us in working with researchers and HackerOne.

RFC, WIP.

**Edit**: Referencing https://github.com/monero-project/monero-site/pull/282 https://github.com/monero-project/meta/issues/39

### CODE REVIEW

File: VULNERABILITY_RESPONSE_PROCESS.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+- Any physical or electronic attempts against Monero community property and/or data centers
+
+## I. Point of Contacts for Security Issues
+
+```
+ric@getmonero.org
+BDA6 BD70 42B7 21C4 67A9  759D 7455 C5E3 C0CD CEB9
+
+luigi1111@getmonero.org
+8777 AB8F 778E E894 87A2  F8E7 F4AC A018 3641 E010
```
Comment (jonathancross) [inline VULNERABILITY_RESPONSE_PROCESS.md]: Note: we should have this uploaded to sks-keyservers.net

File: VULNERABILITY_RESPONSE_PROCESS.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+moneromooo.monero@gmail.com
+48B0 8161 FBDA DFE3 93AD  FC3E 686F 0745 4D6C EFC3
+
+jaquee.monero@gmail.com
+D21E 9CC1 2F51 C4FE A9E0  52FF 384E 52B0 9F45 DC39
```
Comment (jonathancross) [inline VULNERABILITY_RESPONSE_PROCESS.md]: Note: we should have this uploaded to sks-keyservers.net

File: VULNERABILITY_RESPONSE_PROCESS.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
+# Monero Vulnerability Response Process
+
+## Preamble
+
+Researchers/Hackers: while you research/hack, we ask that you please refrain from committing the following:
+- Denial of Service / Active exploiting against the network
+- Social Engineering of Monero staff or contractors
+- Any physical or electronic attempts against Monero community property and/or data centers
+
+## I. Point of Contacts for Security Issues
```
Comment (jonathancross) [inline VULNERABILITY_RESPONSE_PROCESS.md]: "Points of Contact"

File: VULNERABILITY_RESPONSE_PROCESS.md (hunk 1 above)
Comment (anonimal) [inline VULNERABILITY_RESPONSE_PROCESS.md]: The keys should also be in-tree like what monero and kovri does.

File: VULNERABILITY_RESPONSE_PROCESS.md (hunk 2 above)
Comment (anonimal) [inline VULNERABILITY_RESPONSE_PROCESS.md]: The keys should also be in-tree like what monero and kovri does.

File: VULNERABILITY_RESPONSE_PROCESS.md (hunk 3 above)
Comment (anonimal) [inline VULNERABILITY_RESPONSE_PROCESS.md]: Thanks.

File: VULNERABILITY_RESPONSE_PROCESS.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
+# Monero Vulnerability Response Process
+
+## Preamble
+
+Researchers/Hackers: while you research/hack, we ask that you please refrain from committing the following:
+- Denial of Service / Active exploiting against the network
+- Social Engineering of Monero staff or contractors
+- Any physical or electronic attempts against Monero community property and/or data centers
```
Comment (anonimal) [inline VULNERABILITY_RESPONSE_PROCESS.md]: Yes, that seems clearer.

File: VULNERABILITY_RESPONSE_PROCESS.md (hunk 2 above)
Comment (jonathancross) [inline VULNERABILITY_RESPONSE_PROCESS.md]: Found: https://raw.githubusercontent.com/monero-project/monero/master/utils/gpg_keys/jaquee.asc

File: VULNERABILITY_RESPONSE_PROCESS.md (hunk 1 above)
Comment (jonathancross) [inline VULNERABILITY_RESPONSE_PROCESS.md]: Found it... Maybe we should link the key fingerprint?
https://github.com/monero-project/monero/blob/master/utils/gpg_keys/luigi1111.asc

File: VULNERABILITY_RESPONSE_PROCESS.md (hunk 3 above)
Comment (jonathancross) [inline VULNERABILITY_RESPONSE_PROCESS.md]: Still an extra "s", sorry, English is strange...
Should be "Points of Contact" per https://en.wiktionary.org/wiki/points_of_contact#English

File: VULNERABILITY_RESPONSE_PROCESS.md (hunk 1 above)
Comment (anonimal) [inline VULNERABILITY_RESPONSE_PROCESS.md]: Linking would reduce redundancy, but I think we should let them either commit the key to the repo and/or decide where they want their fingerprint to reside.

File: VULNERABILITY_RESPONSE_PROCESS.md (hunk 2 above)
Comment (anonimal) [inline VULNERABILITY_RESPONSE_PROCESS.md]: Linking would reduce redundancy, but I think we should let them either commit the key to the repo and/or decide where they want their fingerprint to reside.

**Pull request merged**